bylaws.md

```diff
@@ -126,25 +115,27 @@ The Fraternity of Lambda Alpha Nu exists to cultivate a strong community around
     3. 2/3rds of the board members must be present to constitute an official board meeting.
     4. Emergency meetings of the Board shall be held when directed by the president or by two members of the voting board.
 	   All Board members must be given notice of such a meeting and the meeting must be conducted in as similar a fashion as possible to a regular board meeting.
-8. Vacancies and Removal:
+9. Vacancies and Removal:
     1. A vacancy in the office of the President shall be filled by the Vice-President of Administration unless he or she is unable to serve, at which time a full special election must be held.
-    2. All vacancies in other Board positions are filled with a special election.
+    2. All vacancies in other elected positions are filled with a special election.
     3. Vacancies in nominated positions shall be filled by nomination of the Board officer to whom the nominated position reports, as approved by a vote of 2/3rds of the Board.
 	   If the Board officer fails to promptly supply a nominee or fails to supply a nominee acceptable to 2/3rds of the Board, 2/3rds of the Board may install a nominee of their choosing.
     4. Any officer, including the President and appointed positions, may be removed from office or suspended from office by a vote of two-thirds of the active membership at a regular meeting.
 	   Removal or suspension may be with or without cause.
 	   The person under consideration shall be given an opportunity to be heard at the meeting with sufficient notice of the proposed removal or suspension.
 	   The person under consideration shall not be counted in determining the presence of a quorum nor whether the required two-thirds vote has been obtained.
 	   Upon a vote of removal or suspension of an officer, the normal vacancy policy immediately comes into effect.
-6. Powers of the Board
-   1. In its executive capacity, the Board is authorized to plan and conduct Rush as well as other events; to set and collect dues; take disciplinary action against Members under established Policies and Codes of Conduct; to accept new Pledges into the Fraternity and conduct all business necessary and proper to the function and continuance of the Fraternity that are otherwise not specified in these bylaws.
-   2. The Board may make and impose no Rule, Policy or Code of Conduct upon the Membership of the Fraternity save by a 2/3rds vote of the Membership taken at a General or Special meeting of the Fraternity.
+10. Powers of the Board
+	1. In its executive capacity, the Board is authorized to plan and conduct Rush as well as other events; to set and collect dues; take disciplinary action against Members under established Policies and Codes of Conduct; to accept new Pledges into the Fraternity and conduct all business necessary and proper to the function and continuance of the Fraternity that are otherwise not specified in these bylaws.
```

Comment:
Structurally, I appreciate that this clause explicitly delineates Board powers.

However, the subclause discussing its disciplinary responsibilities is confusing. Is Standards Board given its disciplinary responsibility as a matter of delegation from the Executive Board, or is Standards Board an "equal" body but with radically different responsibilities?

I prefer the latter interpretation. I'm not sure how exactly to make this clearer, though; perhaps moving the section about discipline to a new clause, that discusses the relationship of Exec and Standards, is for the best.